docs/reference/search/search-your-data/retrievers-examples.asciidoc

```diff
@@ -1523,3 +1522,53 @@ Note that our example applies two reranking steps. First, we rerank the top 100
 documents from the `knn` search using the `my-rerank-model` reranker. Then we
 pick the top 10 results and rerank them using the more fine-grained
 `my-other-more-expensive-rerank-model`.
+
+[discrete]
+[[pinned-retriever]]
+==== Example: Using the pinned retriever
+
+The pinned retriever allows you to always place specific documents at the top of the results, with the remaining hits provided by a secondary retriever.
+
+.Parameters
```

Comment:
I don't think parameters belong here, it should be in the retrievers overview. If you notice no parameters are in any of the other examples. Instead, please add parameter information to [retriever.asciidoc](https://github.com/elastic/elasticsearch/blob/8.19/docs/reference/search/retriever.asciidoc) and follow the paradigm that we use for other retreivers.

Reply:
++ The examples pages should only house examples, not reference info :)